### ISSUE TITLE
Part 2. Workspace and Files. Exercise "setwd".

### ISSUE THREAD

**Issue by rumyantseva** [swirldev/swirl#266]
To change the working directory I typed 

```
setwd("./testdir")
```

and the directory was changed by R. But swirl thought that it isn't correct and expected just setwd("testdir") from me.

So I had to do 

```
setwd("..")
setwd("testdir") 
```

to finish this task.

Do you think you could possible write some explanation for this task?

**Comment by seankross** [comment]
Hi @rumyantseva,

I see what you mean. I'll make the appropriate changes to the lesson.

**Comment by WilCrofter** [comment]
It's probable that the course author used the default test, `omnitest(correctExpr='setwd("testdir")', in an early draft and neglected to generalize it to something like 
```
any_of_exprs('setwd("testdir"), 'setwd("./testdir")')
```
in a later draft.

Perhaps swirl should reset the current working directory in circumstances like this. I'll cross-post the issue to the [swirl 3.0 thread](https://github.com/swirldev/swirl/issues/241.)

**Comment by seankross** [comment]
@WilCrofter I agree that's a good idea for swirl 3.0 and that this question should use `any_of_exprs`. Still, the Workspace and Files lesson should have some explanation about the meaning of `.` and `..` so I'll work that in.

**Comment by WilCrofter** [comment]
On Mon, 23 Feb 2015 09:02:26 -0800
Sean Kross <notifications@github.com> wrote:

> @WilCrofter I agree that's a good idea for swirl 3.0 and that this question should use `any_of_exprs`. Still, the Workspace and Files lesson should have some explanation about the meaning of `.` and `..` so I'll work that in.
> 
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/swirldev/swirl/issues/266#issuecomment-75583720

Right. Thanks, Sean. I'm dashing in and out of swirl at the moment, so my remarks may not be all that coherent =D.

**Comment by pipe0703** [comment]
Hi, I have the same issue - fedora 21, R 3.1.3, RStudio 0.98.1103

> setwd("~/testdir")
| Not quite, but you're learning! Try again. Or, type info() for more options.
| Use setwd("testdir") to set your working directory to "testdir".

**Comment by pam2307** [comment]
hello, i have the same issue m wondering what swirl requires
